npm/qsharp/ux/circuit-vis/circuit.ts

```diff
@@ -10,6 +10,7 @@ export {
   type Component,
   type DataAttributes,
   type Operation,
+  type Measurement,
```

Comment:
I haven't read the rest of the PR so maybe you use this later - but you shouldn't need to export these individual types if you just export `Component` - I don't expect you'll need to refer to those types by name

Reply:
There are some functions throughout that only make sense when operating on Operations or Measurements. Adding controls to gates, for example, can't be done on Measurements.